sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Samples/Sample5_ImportProject.cs

```diff
@@ -15,6 +15,7 @@
 
 namespace Azure.AI.Language.Conversations.Tests.Samples
 {
+    [IgnoreServiceError(429, "429", Reason = "Exceeded rate limit of S pricing tier given number of tests run")]
```

Comment:
this is a very nice little decorator (not sure if that's the right .NET term)!

Reply:
"Attribute", but thanks! I created this a while back for service issues that were not necessarily endemic of real-world usage and probably wouldn't be fixed.